build.gradle

```diff
@@ -116,6 +116,38 @@ subprojects {
             force "jdepend:jdepend:${project.jdepend.toolVersion}"
         }
     }
+
+    apply plugin: 'checkstyle'
+    configurations {
+        checkstyleConfig
+    }
+    def versions = [
+            checkstyle: '6.17',
+    ]
+    dependencies {
+        checkstyleConfig ("com.puppycrawl.tools:checkstyle:${versions.checkstyle}") {
+            transitive = false
+        }
+        checkstyle "com.puppycrawl.tools:checkstyle:${versions.checkstyle}"
+    }
+    checkstyle {
+        config = resources.text.fromArchiveEntry(configurations.checkstyleConfig, 'google_checks.xml')
+        ignoreFailures false
```

Comment:
Yea you're right that it makes sense that ignoreFailures wouldn't be needed with that explicit exception workaround - but I tried removing it or setting it to true and the gradle check task hangs.